Pid.hpp

```diff
@@ -15,11 +17,20 @@ class Pid {
     /** clear any windup */
     void clearWindup();
 
+
+    //tuner stuff
+    void initialize_tuner();
+
+    void set_from_tuner();
+
+    void start_cycle();
```

Comment:
It would be best to somehow denote these functions have to do with the autoTuner like startAutoTunerCycle. Also some documentation on these functions would be useful.